se.sjoerd.GIScan.yml

```diff
@@ -0,0 +1,43 @@
+app-id: se.sjoerd.GIScan
+runtime: org.kde.Platform
+runtime-version: '5.15-21.08'
+sdk: org.kde.Sdk
+base: com.riverbankcomputing.PyQt.BaseApp
+base-version: "5.15-21.08"
+command: runner.sh
+finish-args:
+  - --share=ipc
+  - --socket=x11
+  - --share=network
+  - --filesystem=xdg-documents
+  - --device=dri
+  
+modules:  
+
+  - python3-requirements.json
+  - python3-matplotlib.json
+  - python3-scipy.json
+        
+  - name: GIScan
+    only-arches: x86_64
+    sources:
+      - type: git
+        url: https://github.com/SjoerdB93/GIScan.git
+        commit: 9a5a7a4804bc81848895485bfd41fc3f5ff401e8
+
+    buildsystem: simple
+    build-commands:
+      - install -D se.sjoerd.GIScan.desktop /app/share/applications/se.sjoerd.GIScan.desktop
```

Comment:
the desktop fie is lacking caterogies wich is important to determine in which menu the entry is (on some DE)

Reply:
Thanks, haven't seen this categories since pre-Unity Ubuntu so never considered them. But you're rightt, some DE's still use these!